src/servers/database.ts

```diff
@@ -822,6 +823,93 @@ export class DatabaseServer extends SkeletonServer {
     return true;
   }
 
+  protected async _updatePollBlock(filterID_: Buffer): Promise<void> {
+    const latestBlock = await this._fetchCurrentBlockID();
+    await this._query(
+      'UPDATE filter SET poll_block = $1 + 1 WHERE uuid_send(id) = $2',
+      [latestBlock, filterID_]
+    );
+  }
+
+  protected async _getFilterChangesEvent(
+    filter: {
+      addresses: string;
+      poll_block: web3.Quantity;
+      to_block?: web3.Quantity;
+      topics?: web3.FilterTopic[];
+      type: 'block' | 'event' | 'transaction';
+    },
+    filterID_: Buffer
+  ): Promise<web3.LogObject[]> {
+    const where = [];
+
+    const fromBlock = parseBlockSpec(filter.poll_block);
```

Comment:
Due to the fact that in the previous version we had such logic:
`coalesce(f.poll_block, f.from_block, 0)`

https://github.com/aurora-is-near/aurora-relayer/blob/39cbe9e05d557e064e263b14851467486e5ecf2d/etc/schema/functions/eth_getFilterChanges.sql#L40

At the time of filter creation we set `poll_block` to the latest block:
https://github.com/aurora-is-near/aurora-relayer/blob/39cbe9e05d557e064e263b14851467486e5ecf2d/etc/schema/functions/eth_newFilter.sql#L12

and `f.from_block` never been used

We can also add to our docs something like this:
```
eth_getFilterChanges only returns logs since the filter was created, regardless of the block passed in to create the filter.
```